bot/resources/tags/keyerror.md

```diff
@@ -0,0 +1,17 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary. \
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
```

Comment:
I believe there is one too many spaces, since there is one on this line and one on the previous line before the backslash.